### PR TITLE
[Spring Apps] Add support for the custom container.

### PR DESCRIPTION
**Pull request by LGDoor**
**Task name**: AzureSpringCloud

**Description**: Add support for the custom container.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N)  Y

**Attached related issue:** (Y/N) N
**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by LGDoor** [comment]
@mmrazik Could you please help review this?

**Comment by max-zaytsev** [comment]
@LGDoor Did you E2E test your changes?

File: Tasks/AzureSpringCloudV0/task.loc.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
   "author": "Microsoft Corporation",
   "preview": true,
   "version": {
-    "Major": 0,
```
Comment (max-zaytsev) [inline Tasks/AzureSpringCloudV0/task.loc.json:20]: Extra space

**Review by max-zaytsev — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Review by DmitriiBobreshev — APPROVED** [review]

**Review by mmrazik — APPROVED** [review]

**Comment by LGDoor** [comment]
> @LGDoor Did you E2E test your changes?

Yes, already tested.

**Pull request merged**